inc/helpers.php

```diff
@@ -23,7 +23,18 @@ function primer_get_the_page_title() {
 
 		case is_front_page() :
 
-			$title = get_the_title( get_option( 'page_on_front' ) );
+			/**
+			 * Filter the page title when the front page is set to display latest posts.
+			 *
+			 * Settings > Reading > Front page displays > Your latest posts
+			 *
+			 * @since NEXT
+			 *
+			 * @var string
+			 */
+			$latest_posts_title = (string) apply_filters( 'primer_latest_posts_title', '' );
```

Comment:
@EvanHerman I kind of like the idea of _not_ displaying any page title in this scenario by default. Keeping the filter in place allows anyone to still change it. Thoughts?